tools/bridge/bridge/main.py

```diff
@@ -234,6 +235,40 @@ def make_validator_balance_watcher(config, control_queue):
     )
 
 
+public_config_keys = (
+    "foreign_rpc_url",
+    "home_rpc_url",
+    "foreign_chain_max_reorg_depth",
+    "home_chain_max_reorg_depth",
+    "foreign_chain_token_contract_address",
+    "foreign_bridge_contract_address",
+    "home_bridge_contract_address",
+    "foreign_chain_event_fetch_start_block_number",
+    "home_chain_event_fetch_start_block_number",
+)
+
+
+def make_webservice(*, config, recorder):
+    d = config["webservice"]
+    if d["enabled"]:
+        ws = Webservice(host=d["host"], port=d["port"])
+    else:
+        ws = DummyWebservice()
```

Comment:
this isn't really a webservice at all. Calling `enable_internal_state` on it does nothing and is confusing. Returning `None` if no webserver should run would be much cleaner.

Reply:
ok, I will change that.